ci/integration-tests-windows.groovy

```diff
@@ -86,8 +89,7 @@ node('py36') {
                                 python scripts/plugin/package_plugin.py; \
                                 cd python/lib/dcoscli; \
                                 make env; \
-                                rm -f ./env/Scripts/dcos.exe; \
-                                python -c 'import urllib.request; urllib.request.urlretrieve(\\\"https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe\\\", \\\"dist/dcos.exe\\\")'; \
+                                mv ../../../dcos.exe dist; \
```

Comment:
Did you try to run `wget` directly from this node? It has winbash installed and they might have `wget` there.

Reply:
```
+ wget -O dist/dcos.exe https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe
--17:41:18--  https://downloads.dcos.io/cli/testing/binaries/dcos/windows/x86-64/master/dcos.exe
           => `dist/dcos.exe'
Resolving downloads.dcos.io... done.
Connecting to downloads.dcos.io[99.84.239.15]:443... connected.

Unable to establish SSL connection.

Unable to establish SSL connection.
```